### PR TITLE
fix 4847: Batch update rest api V2 fails

### PR DESCRIPTION
**Pull request by bartcharbon**
#### Checklist
- [x] Unit tests
- [ ] Updated testplan

### CODE REVIEW

File: molgenis-data-validation/src/test/java/org/molgenis/data/validation/RepositoryValidationDecoratorTest.java [hunk 1]
```diff
@@ -2010,6 +2014,33 @@ public void update()
 	}
 
 	@Test
```
Comment (Mark-de-Haan) [inline molgenis-data-validation/src/test/java/org/molgenis/data/validation/RepositoryValidationDecoratorTest.java]: You can set a test to expect an exception:

`@Test(expectedExceptions = { MolgenisDataException.class }, expectedExceptionsMessageRegExp = "The entity you are trying to update [id999] does not exist.")`

Comment (fdlk) [inline molgenis-data-validation/src/test/java/org/molgenis/data/validation/RepositoryValidationDecoratorTest.java]: But then the regex probably should escape the `[` ;-)

Comment (Mark-de-Haan) [inline molgenis-data-validation/src/test/java/org/molgenis/data/validation/RepositoryValidationDecoratorTest.java]: True :p

Comment (bartcharbon) [inline molgenis-data-validation/src/test/java/org/molgenis/data/validation/RepositoryValidationDecoratorTest.java]: I know, however I find the current solution clearer to read

**Comment by tommydeboer** [comment]
The new exception message is only thrown when you update a whole entity, not when you do an attribute specific update:

Calling ```/v2/org_molgenis_test_TypeTest/xdatetime``` with an invalid id gives:

```java
{
  "errors": [
    {
      "message": "Operation failed. Identifier: 77 is not valid"
    }
  ]
}
```

**Pull request merged**